collectors/0/goproc.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import traceback
+
+import requests
+import sys
+import time
+
+from collectors.etc import goprocconf
+from collectors.lib import utils
+
+COLLECTION_INTERVAL_SECONDS = 15
+SKIP_MEM_KEYS = ['PauseNs', 'PauseEnd', 'BySize']
+
+
+# "stats":
+#     {"advanced":
+#          {"closed": 748,
+#           "opened": 0
+#          },
+#      "connections": 748
+#      }
+
+def _print_nested_metrics(key, value, ts, root=None):
+    if isinstance(value, dict):
+        for ck, cv in value.items():
+            _print_nested_metrics(ck, cv, ts, ('' if root is None else (root + '.')) + key)
+    else:
+        print("%s %s %s" % (key, ts, value) if root is None else "%s.%s %s %s" % (root, key, ts, value))
+
+
+def main(argv):
+    expvar_url = goprocconf.get_expvar_url()
```

Comment:
First line of main is usually `utils.drop_privileges()`

Reply:
Thanks, added